src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateDrawerForm.tsx

```diff
@@ -253,6 +258,12 @@ const useCreateDrawerForm = (
         vmDraft.spec.template.spec.volumes = isRHELTemplate(processedTemplate)
           ? updateCloudInitRHELSubscription(updatedVolumes, subscriptionData, autoUpdateEnabled)
           : updatedVolumes;
+
+        if (useUDN) {
+          const defaultInterface = getInterfaces(vmDraft)?.[0];
```

Comment:
The default interface can be in a different position, can you please improve this?